### PR TITLE
Display internal case name

### PR DESCRIPTION
**Pull request by DavidJayakumar**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/EUI-8981

### Change description ###
Display internal case name instead of public name

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [x] Does this PR introduce a breaking change

### CODE REVIEW

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by udaydenduluri33 — APPROVED** [review]

File: src/hearings/components/hearing-parties-title/hearing-parties-title.component.ts [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
+import { AppConstants } from '../../../app/app.constants';
 import { ServiceHearingValuesModel } from '../../models/serviceHearingValues.model';
 import * as fromHearingStore from '../../store';
 
 @Component({
   selector: 'exui-hearing-parties-title',
   templateUrl: './hearing-parties-title.component.html'
 })
-export class HearingPartiesTitleComponent implements OnInit, OnDestroy {
+export class HearingPartiesTitleComponent implements OnInit {
```
Comment (connorpgpmcelroy) [inline src/hearings/components/hearing-parties-title/hearing-parties-title.component.ts:13]: Why has OnDestroy been removed here yet is still present?

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM apart from comment

File: src/hearings/components/hearing-parties-title/hearing-parties-title.component.ts (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply src/hearings/components/hearing-parties-title/hearing-parties-title.component.ts]: @connorpgpmcelroy - I added it back as it is required. Thank you for finding this one.

**Pull request merged**